packages/api/src/Base.ts

```diff
@@ -107,7 +107,6 @@ export default abstract class ApiBase<CodecResult, SubscriptionResult> implement
     const thisProvider = options.source
       ? options.source._rpcBase._provider.clone()
       : options.provider;
-
```

Comment:
keep these empty lines. Jaco said it's a code style.

Reply:
done